packages/server/type-augmentations/true-myth.d.ts

```diff
@@ -0,0 +1,10 @@
+// Only need to do this because our CommonJS app does not support true-myth's export maps
```

Comment:
another reason to move to an ESM build ASAP - libraries are relying on "exports" maps in package.json more and more, and we can't use them (at least their TS types)